python/dlisio/plumbing/wellref.py

```diff
@@ -51,14 +50,15 @@ class Wellref(BasicObject):
         'PERMANENT-DATUM-ELEVATION' : scalar('permanent_datum_elevation'),
         'ABOVE-PERMANENT-DATUM'     : scalar('above_permanent_datum'),
         'MAGNETIC-DECLINATION'      : scalar('magnetic_declination'),
-        'COORDINATE-1-NAME'         : dictentry('coordinate'),
-        'COORDINATE-1-VALUE'        : dictentry('coordinate'),
-        'COORDINATE-2-NAME'         : dictentry('coordinate'),
-        'COORDINATE-2-VALUE'        : dictentry('coordinate'),
-        'COORDINATE-3-NAME'         : dictentry('coordinate'),
-        'COORDINATE-3-VALUE'        : dictentry('coordinate'),
+        'COORDINATE-1-NAME'         : custom(),
+        'COORDINATE-1-VALUE'        : custom(),
+        'COORDINATE-2-NAME'         : custom(),
+        'COORDINATE-2-VALUE'        : custom(),
+        'COORDINATE-3-NAME'         : custom(),
+        'COORDINATE-3-VALUE'        : custom(),
```

Comment:
I know it was my name, but I'm not so sure that I like it! Maybe 'manual', 'later' or 'no_op'?